_sass/module/_election-header.scss

```diff
@@ -5,9 +5,17 @@
 @include grid-media($neat-grid--medium) {
   .election-header__day {
     float: left;
+    color: $color-navy;
+    font-size: $h3-font-size;
+    font-weight: bold;
   }
 
   .election-header__updated-at {
     float: right;
   }
+
+  .election-header__title {
+    color: $color-navy;
+    font-weight: 400;
+  }
```

Comment:
I think the color and font style changes shouldn't be restricted to medium+ screens, shouldn't they apply to all screen sizes?

Reply:
Got it, removed the screen size restriction.